templates/partials/tasks_list.html

```diff
@@ -1,5 +1,5 @@
 <div id="tasks-list" class="col-sm-4 inbox-items left-pane">
-  <ul ng-show="!loading && !error">
+  <ul ng-show="!loading && !error && tasks.length">
```

Comment:
This isn't strictly necessary I don't think, but I figured it helps differentiate between the different states